client-projects/the-lumery/cardImages.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  CARD_IMAGE_SELECTOR,
+  CARD_LINK_SELECTOR,
+  IMAGE_CONTAINER_SELECTOR,
+} from "./selectors";
+import { prefersReducedMotion } from "./utils";
+
+export function setUpImageAnimation() {
+  const cards = document.querySelectorAll(CARD_LINK_SELECTOR);
+
+  cards.forEach((card) => {
+    const imageContainer = card.querySelector(IMAGE_CONTAINER_SELECTOR);
+    if (!(imageContainer instanceof HTMLDivElement)) {
```

Comment:
That's a nice check.

Reply:
yeah, I had been finding myself adding an `: HTMLDivElement | null` annotation to `const imageContainer` in order to use `imageContainer.style`, and then _also_ checking `if (imageContainer)`... and I remembered there's a way I could achieve both in one check!

Reply:
Yeah, makes sense. I'll have to remember this one.